test/recipes/80-test_ssl_old.t

```diff
@@ -78,10 +86,20 @@ my $client_sess="client.ss";
 # If you're adding tests here, you probably want to convert them to the
 # new format in ssl_test.c and add recipes to 80-test_ssl_new.t instead.
 plan tests =>
-    1				# For testss
-    +5  			# For the first testssl
+    1				# For fipsinstall
+    +1				# For testss
+    +5  			# For the testssl with default provider
+    +5  			# For the testssl with fips provider
     ;
 
+ok(run(app(['openssl', 'fipsinstall',
+            '-out', bldtop_file('providers', 'fipsinstall.cnf'),
+            '-module', bldtop_file('providers', platform->dso('fips')),
+            '-provider_name', 'fips', '-mac_name', 'HMAC',
+            '-macopt', 'digest:SHA256', '-macopt', 'hexkey:00',
+            '-section_name', 'fips_sect'])),
+   "fipsinstall");
```

Comment:
Considering how this is copied all over the place, we might want to consider making this a separate script.

Reply:
Yes...and a separate "make test" dependency so that we don't have to do this everywhere? Not *this* PR though I think.

Reply:
Oh!  That too

Reply:
Yeah, I'm getting to a point where I'll do this, in a separate PR